### PR TITLE
Added index.html and styles.css files

### PR DESCRIPTION
**Pull request by ndayishimiyeeric**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Survey | Form</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="header">
+        <h1 id="title">Microverse Trials session</h1>
+        <p id="description">this session is done by Eric and Toka</p>
+      </header>
```
Comment (ndayishimiyeeric) [inline index.html:15]: Please you should use the proper way of semantic tags. the  `<header>` should be above the `<div>` tag please you should change that.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+            id="number"
+            min="20"
+            max="100"
+            class="form-control"
+            placeholder="Age"
+          />
+        </div>
+        <div class="form-control">
+          <p>What is your current role?</p>
+          <select id="dropdown" name="role" class="form-control" required>
```
Comment (ndayishimiyeeric) [inline index.html:53]: You should use proper semantic tag to describe  title like `<h1>`, `<h2>` ... `<6>` .

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+.form-group {
+  margin: 0 auto 0.5rem auto;
+  padding: 0.25rem;
+}
+.form-control {
+  display: block;
+  width: 100%;
+  /* height: 0rem; */
+  padding: 0.5rem 1rem;
```
Comment (ndayishimiyeeric) [inline styles.css:73]: You should not comment unwanted codes because it makes your code unclean. Please remove that

**Review by ndayishimiyeeric — COMMENTED** [review]
Changes required ♻️ 

Please remember to use the Readme.md file to provide description 📔  of your project correctly

Please use the linters file configuration

Please remember to use .gitignore file to ignore unwanted file like node_modules. 

Please you should provide a well descriptive message for your PR

Additional changes 🏳️

File: index.html (hunk 1 above)
Comment (sja-thedude) [reply index.html:15]: Good job 👍 in identifying the proper use of semantic tags.

File: index.html (hunk 2 above)
Comment (sja-thedude) [reply index.html:53]: Good job 👍 in identifying the correct heading tag for headings.

File: styles.css (hunk 3 above)
Comment (sja-thedude) [reply styles.css:73]: Optional: Good job 👍 but You should ask them to remove it.

**Review by sja-thedude — APPROVED** [review]
Hi @ndayishimiyeeric, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ndayishimiyeeric** [comment]
Thank you @sja-thedude

**Pull request merged**